images/base/files/usr/local/bin/mount-product-files

```diff
@@ -22,22 +22,33 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# Explicitly set PATH so as not to be inherited from the container
+# We have no reason to be using any binary from the container, and the
+# container PATH may lack normal "host" (kind node container in this case)
+# system paths.
+#
+# See: https://github.com/kubernetes-sigs/kind/issues/2551
+#
+# All of the binaries this script needs are in /usr/bin currently, but this is
+# the full normal PATH for this image with pretty standard linux paths.
+export PATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
+
 # The bundle represents the dir path to container filesystem, container runtime state [1] is
 # passed to the hook's stdin
 #
 # [1] https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#state
 #
 bundle=$(jq -r .bundle)
 
-cp /kind/product_* "$bundle/rootfs/"
+cp /kind/product_* "${bundle:?}/rootfs/"
```

Comment:
inconsistent quote location here but /shrug

at some point I will find a linter for this and normalize the whole repo

Reply:
TIL
```
echo ${test:?}
bash: test: parameter null or not set
```